packages/tests/tests/filter/single_node/subscribe.node.spec.ts

```diff
@@ -220,11 +220,14 @@ describe("Waku Filter V2: Subscribe", function () {
     });
   });
 
-  it("Subscribe to 100 topics at once and receives messages", async function () {
-    let topicCount = 30;
+  it("Subscribe to 100 topics (new limit) at once and receives messages", async function () {
```

Comment:
Should we make a todo to remove old tests?
Are there any other tests that might need to be changed/removed?

Reply:
Added TODO
No, only those tests are affected afaik